src/server/static/js/lines.js

```diff
@@ -171,387 +60,264 @@ function updateInfectionTotals() {
-      const scenarios = getListOfScenarios(activeData);
-      const hoveredValues = scenarios.map(s => {
-        return activeData[s][diffDays];
-      });
-      tooltip
-        .style("opacity", 1)
-        .html(hoveredValues.map((h, i) => `<span class="color${i+1}">${(h/10).toFixed(5)}</span>`).join("<br>")) //TODO: `<span class="color${i}">h</span>` // scenarios[i] + ": " +
-        .style("left", d3.event.pageX + "px")
-        .style("top", d3.event.pageY - 28 + "px");
-    });
+var plotlyConfig = {
+  displaylogo: false,
+  responsive: true,
+  scrollZoom: false
+};
 
-  //initialization
-  update();
-  d3.select("#selectButton").property("value", selected.country);
-  // update the containment measures with the new selected country
-  var countryName = listOfCountries.find(c => c.key == selected.country).value
-  // update_containment_measures(countryName);
-  update_country_in_text(countryName);
-  updateInfectionTotals()
-
-  // A function that update the chart
-  function update() {
-    yDomain[1] = getMaxYValueForCountry(
-      infectedPer1000.mitigations,
-      selected.country
-    ) / 10;
-    y = y.domain(yDomain);
-    function updateLine(myLine, lineData) {
-      myLine
-        .datum(lineData)
-        .transition()
-        .duration(1000)
-        .attr(
-          "d",
-          d3
-            .line()
-            .x(function (d, i) {
-              return x(d3.timeDay.offset(new Date(), i));
-            })
-            .y(function (d) {
-              return y(+d/10);
-            })
-        );
-    }
+Plotly.newPlot(plotyGraph, [], layout, plotlyConfig);
 
-    yAxis
-      .transition()
-      .duration(1000)
-      .call(d3.axisLeft(y).ticks(10));
 
-    for (l in lines) {
-      updateLine(lines[l], activeData[l]);
-    }
+function getMaxYValueForRegion(mitigations) {
+  var highestVals = [];
+  Object.values(mitigations).forEach(mitigation => {
+    Object.values(mitigation).forEach(values => {
+      highestVals.push(Math.max(...values));
+    });
+  });
+  return Math.max(...highestVals);
+}
 
-    updateInfectionTotals()
-  }
 
-  // When the button is changed, run the updateChart function
-  d3.select("#selectButton").on("change", function (d) {
-    // recover the option that has been chosen
-    selected.country = d3.select(this).property("value");
-    // change url param
-    setGetParam("selection", selected.country);
-    infectedPer1000 = data.regions[selected.country].data.infected_per_1000;
-    activeData = infectedPer1000.mitigations[selected.mitigation];
-
-    var countryName = listOfCountries.find(c => c.key == selected.country).value
-    // run the updateChart function with this selected option
-    update();
-    // update the containment measures with the new selected country
-    // update_containment_measures(countryName);
-    // update the name of the country in the text below the graph
-    update_country_in_text(countryName);
```

Comment:
Removing this (or changing some parts) broke the JS fill in the text below the graph. 
![image](https://user-images.githubusercontent.com/2741256/77225875-c3e1fd00-6b73-11ea-9e00-92563a54c077.png)

Reply:
https://github.com/epidemics/covid/issues/217